case-lib/lib.sh

```diff
@@ -571,6 +575,34 @@ func_lib_check_sudo()
     }
 }
 
+func_lib_enable_pipewire()
+{
+    dlogi "Starting Pipewire..."
+    
+    sudo systemctl --global unmask pipewire{,-pulse}.{socket,service}
+    systemctl --user unmask pipewire{,-pulse}.{socket,service}
+    systemctl --user start pipewire{,-pulse}.{socket,service}
+    sudo systemctl --global unmask wireplumber.service 
+    systemctl --user unmask wireplumber.service
+    systemctl --user start wireplumber.service
+    systemctl --user daemon-reexec
+    systemctl --user daemon-reload
+
+    dlogi "Pipewire started"
+}
+
+func_lib_disable_pipewire()
+{
+    dlogi "Stopping Pipewire..."
+
+    systemctl --user stop wireplumber.service
+    sudo systemctl --global mask wireplumber.service
+    systemctl --user stop pipewire{,-pulse}.{socket,service}
+    sudo systemctl --global mask pipewire{,-pulse}.{socket,service}
```

Comment:
Same as above, I would stop everything first, and then mask.

Mmm... or maybe both at the same time in case there is a race where a client causes a restart between stopping and masking?